guides/common/modules/proc_enabling-puppet.adoc

```diff
@@ -0,0 +1,54 @@
+:_module-type: PROCEDURE
+
+[id="proc_enabling-puppet_{context}"]
```

Comment:
```suggestion
[id="Enabling_Puppet_Integration_{context}"]
```

We usually set the anchor to the title + `_{context}`. Unfortunately, we cannot use two attributes in anchors.